packages/frontend-2/pages/projects/[id]/index.vue

```diff
@@ -268,6 +276,8 @@ const pageTabItems = computed((): LayoutPageTabItem[] => {
   return items
 })
 
+const isAccEnabled = ref(true) // TODO
```

Comment:
what should this be?

Reply:
retrieving from feature flag now. resolved